addons/website_forum/views/website_forum.xml

```diff
@@ -218,6 +226,19 @@
     </t>
 </template>
 
+<template id="website.403">
```

Comment:
As per the spec we should display the different message for admin and other users that's why we overwrite it. If we create another one then beacuse of access rule the '403: Forbidden' comes for other users.